### ISSUE TITLE
I can't repeat the result as your paper, on the task of NER?

### ISSUE THREAD

**Issue by milk2we** [dmis-lab/biobert#105]
My config file is that:
[Data]
data_dir = ./Data/NCBI-disease-IOB/
model_type = bert
model_name_or_path=./PretrainModel/biobert_v1.1_pubmed/pytorch
output_dir = ./output/NCBI/
labels = ./Data/NCBI-disease-IOB/labels.txt
max_seq_length = 128

[Optimization]
learning_rate = 5e-5
weight_decay = 0.0
adam_epsilon = 1e-8
max_grad_norm = 1.0
seed = 45

[Train]
no_cuda = False
local_rank = -1
num_train_epochs = 10.0
max_steps = -1
warmup_steps = 0
logging_steps = 500
save_steps = 500
overwrite_output_dir = True
overwrite_cache = True
do_train = True
do_eval = True
do_predict = True
evaluate_during_training = True
eval_all_checkpoints = True
per_gpu_train_batch_size = 32
per_gpu_eval_batch_size = 8
gradient_accumulation_steps = 1

###
Besides, I coding in Pytorch.

**Comment by jhyuklee** [comment]
Hi, we have implemented the PyTorch version of BioBERT in https://github.com/dmis-lab/biobert-pytorch.
